lowpass_filter_design/README.md

```diff
@@ -0,0 +1,30 @@
+# Lowpass Filter Generation
+
+This code generates the lowpass filter to avoid aliasing for audio downsampled to 16kHz. Therefore the cutoff frequency needs to be slightly larger than 8 kHz.
```

Comment:
why 8kHz?  a brief explanation would be nice.

Reply:
Fixed in ddfbcfa5fa241632362249f7bf4b7a5c94ae2d21